src/open-url.coffee

```diff
@@ -101,11 +98,13 @@ module.exports = (robot) ->
 	###
 	# (?:\W(\w+))? match up to the first word after open, capturing just the word
 	robot.respond /open(?:\W(\w+))?/i, (context) ->
-		context.send(context.random(holdings))
-		Promise.try ->
+		context.send(personality.buildMessage('holding', 'greeting'))
+		try
 			get(getBookmarkFromContext(context))
-		.then(-> context.send(context.random(confirmations)))
-		.catch(createErrorReporter(context))
+			.then(-> context.send(personality.buildMessage('confirm', 'pleasantry')))
+			.catch(createErrorReporter(context))
+		catch error
+			createErrorReporter(context)(error)
```

Comment:
This reverts the `Promise.try` improvement, which was a fix from a previous review. Why?